mtbf_job_runner.py

```diff
@@ -380,4 +384,13 @@ def run(self):
 
 if __name__ == '__main__':
     mjr = MtbfJobRunner()
+
+    ## use default settings if no assigned
+    settings_argv = False
```

Comment:
Highly recommend not to modify sys argv here since we have full control of the stack.